packages/api/src/schema/api-schema.yaml

```diff
@@ -1931,9 +2152,78 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/error"
+    delete:
+      tags:
+        - Stream
+      summary: Delete a stream
+      description: >
+        DELETE /stream/{id} deletes an existing stream object.
+
+
+        This will also suspend any active stream sessions, so make sure to wait
+        until the stream has finished. To explicitly interrupt an active
+        session, consider instead updating the suspended field in the stream
+        using the PATCH stream API.
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          description: ID of the stream
+          required: true
+      responses:
+        "204":
+          description: Success (No content)
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+  "/stream/{id}/record":
+    patch:
+      tags:
+        - Stream
+      summary: Patch Recording Toggle
+      description: >
+        PATCH /stream/{id}/record can only modify a stream object. You cannot
+        modify the record value on a session object or the historic stream
+        objects with a parentId (stream objects representing a single live
+        stream session).
+
+
+        The record value is inherited by all future child session objects. Child
+        session objects are read-only.
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          description: ID of the stream
+          required: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/update-record-status"
+      responses:
+        "204":
+          description: Success (No content)
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
```

Comment:
Actually, we should not document this API (sorry if I probably said otherwise). This has been replaced by the single `PATCH /stream/:id` which you documented above so we don't need this here occupying valuable docs space.

Also remember to delete the `update-record-status` schema above when you delete this